tests/test_enterprise_kafka.py

```diff
@@ -9,6 +9,7 @@
 FIXTURES_DIR = os.path.join(CURRENT_DIR, "fixtures", "debian", "enterprise-kafka")
 HEALTH_CHECK = """bash -c 'cp /etc/kafka/kafka.properties /tmp/cub.properties \
                   && echo security.protocol={security_protocol} >> /tmp/cub.properties \
+                  && echo "metric.reporters=" >> /tmp/cub.properties \
```

Comment:
What's the reason for this?

Reply:
This is using `/etc/kafka/kafka.properties` as the properties for running `cub kafka-ready`. But the metrics reporter `io.confluent.metrics.reporter.ConfluentMetricsReporter` configured in `kafka.properties` is not in the CLASSPATH for `kafka-ready`. I assumed that was a test issue, so reset reporter in the properties used to run `cub kafka-ready`.
```
org.apache.kafka.common.KafkaException: Failed create new KafkaAdminClient
        at org.apache.kafka.clients.admin.KafkaAdminClient.createInternal(KafkaAdminClient.java:322)
        at org.apache.kafka.clients.admin.AdminClient.create(AdminClient.java:60)
        at io.confluent.admin.utils.ClusterStatus.isKafkaReady(ClusterStatus.java:138)
        at io.confluent.admin.utils.cli.KafkaReadyCommand.main(KafkaReadyCommand.java:150)
Caused by: org.apache.kafka.common.KafkaException: io.confluent.metrics.reporter.ConfluentMetricsReporter ClassNotFoundException exception occurred
        at org.apache.kafka.common.config.AbstractConfig.getConfiguredInstances(AbstractConfig.java:288)
        at org.apache.kafka.common.config.AbstractConfig.getConfiguredInstances(AbstractConfig.java:263)
        at org.apache.kafka.clients.admin.KafkaAdminClient.createInternal(KafkaAdminClient.java:289)
        ... 3 more
Caused by: java.lang.ClassNotFoundException: io.confluent.metrics.reporter.ConfluentMetricsReporter
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        at java.lang.Class.forName0(Native Method) 
        at java.lang.Class.forName(Class.java:348)
        at org.apache.kafka.common.utils.Utils.newInstance(Utils.java:300)
```